ui/src/scenes/Schema/Table/index.tsx

```diff
@@ -111,65 +120,80 @@ const Table = ({
   const [columns, setColumns] = useState<QuestDB.Column[]>()
 
   useEffect(() => {
-    if (expanded) {
-      combineLatest(
-        from(quest.showColumns(name)).pipe(startWith(null)),
-        of(true).pipe(delay(1000), startWith(false)),
-      ).subscribe(([response, loading]) => {
-        if (response && response.type === QuestDB.Type.DQL) {
-          setColumns(response.data)
-          setLoading(false)
-        } else {
-          setLoading(loading)
-        }
-      })
-    }
-  }, [expanded, refresh, quest, name])
-
-  const handleClick = useCallback(
-    (e) => {
-      onChange(expanded ? "" : name)
+    combineLatest(
+      from(quest.showColumns(name)).pipe(startWith(null)),
+      of(true).pipe(delay(1000), startWith(false)),
+    ).subscribe(([response, loading]) => {
+      if (response && response.type === QuestDB.Type.DQL) {
+        setColumns(response.data)
+        setLoading(false)
+      } else {
+        setLoading(loading)
+      }
+    })
+  }, [refresh, quest, name])
+
+  const tree: TreeNode[] = [
+    {
+      name,
+      kind: "table",
+      render({ toggleOpen }) {
+        return (
+          <ContextMenuTrigger id={name}>
+            <Title
+              description={description}
+              kind="table"
+              name={name}
+              onClick={() => toggleOpen()}
+              partitionBy={partitionBy}
+              suffix={loading && <Loader size="18px" />}
+              tooltip={!!description}
+            />
+          </ContextMenuTrigger>
+        )
+      },
+
+      children: [
```

Comment:
`children` array here currently contains only one descriptor for `Columns` folder, but here we can easily add more, for `Symbols` or any other items.